pkg/sql/zone_config.go

```diff
@@ -41,6 +42,18 @@ func init() {
 
 var errNoZoneConfigApplies = errors.New("no zone config applies")
 
+const secondaryTenantsZoneConfigsEnabledSettingName = "sql.zone_configs.allow_for_secondary_tenant.enabled"
```

Comment:
Need to call out the experimental nature, how about "sql.defaults.experimental_multitenant_zone_configs.enabled" instead? And putting it next to the other experimental sql cluster settings over at pkg/sql/exec_util.go?

Reply:
Talked offline, changing it to `sql.zone_configs.experimental_allow_secondary_tenants.enabled`